test/e2e_node/remote/utils.go

```diff
@@ -43,6 +60,19 @@ func installCNI(host, workspace string) error {
 	if output, err := SSH(host, "sh", "-c", cmd); err != nil {
 		return fmt.Errorf("failed to install cni plugin on %q: %v output: %q", host, err, output)
 	}
+
+	// The added CNI network config is not needed for kubenet. It is only
+	// used when testing the CNI network plugin, but is added in both cases
+	// for consistency and simplicity.
+	glog.V(2).Infof("Adding CNI configuration on %q", host)
+	cniConfigPath := filepath.Join(workspace, cniConfDirectory)
+	cmd = getSSHCommand(" ; ",
+		fmt.Sprintf("mkdir -p %s", cniConfigPath),
+		fmt.Sprintf("echo %s > %s", quote(cniConfig), filepath.Join(cniConfigPath, "mynet.conf")),
```

Comment:
Actually, I feel like it's better to put cni binary and configuration into the tarball in https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/remote/remote.go#L37.

And they will be unpacked to the test node automatically. I can send a PR to do that if you don't have time. :)